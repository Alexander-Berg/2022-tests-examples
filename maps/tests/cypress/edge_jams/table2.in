{"tag"="incomplete";"persistent_id"=5681202146741528770u;"segment_index"=1u;"travel_time"=1.5;"time"="20200528T000400";"signals_accounted"=2u;"last_signal_time"="20200528T000303.486000";};
{"tag"="incomplete";"persistent_id"=5681202146741528770u;"segment_index"=2u;"travel_time"=1.5;"time"="20200528T000400";"signals_accounted"=2u;"last_signal_time"="20200528T000304.486000";};
{"tag"="incomplete";"persistent_id"=5681202146741528770u;"segment_index"=3u;"travel_time"=1.5;"time"="20200528T000400";"signals_accounted"=2u;"last_signal_time"="20200528T000305.486000";};
{"tag"="incomplete";"persistent_id"=5681202146741528770u;"segment_index"=1u;"travel_time"=1.5;"time"="20200528T000430";"signals_accounted"=2u;"last_signal_time"="20200528T000403.486000";};
{"tag"="incomplete";"persistent_id"=5681202146741528770u;"segment_index"=2u;"travel_time"=1.5;"time"="20200528T000430";"signals_accounted"=2u;"last_signal_time"="20200528T000404.486000";};
{"tag"="incomplete";"persistent_id"=5681202146741528770u;"segment_index"=3u;"travel_time"=1.5;"time"="20200528T000430";"signals_accounted"=2u;"last_signal_time"="20200528T000405.486000";};
{"tag"="complete";"persistent_id"=5681202146741528770u;"segment_index"=0u;"travel_time"=1.5;"time"="20200528T000400";"signals_accounted"=2u;"last_signal_time"="20200528T000303.486000";};
{"tag"="complete";"persistent_id"=5681202146741528770u;"segment_index"=1u;"travel_time"=1.5;"time"="20200528T000400";"signals_accounted"=2u;"last_signal_time"="20200528T000303.486000";};
{"tag"="complete";"persistent_id"=5681202146741528770u;"segment_index"=2u;"travel_time"=1.5;"time"="20200528T000400";"signals_accounted"=2u;"last_signal_time"="20200528T000304.486000";};
{"tag"="complete";"persistent_id"=5681202146741528770u;"segment_index"=3u;"travel_time"=1.5;"time"="20200528T000400";"signals_accounted"=2u;"last_signal_time"="20200528T000305.486000";};
{"tag"="complete";"persistent_id"=5681202146741528770u;"segment_index"=0u;"travel_time"=1.5;"time"="20200528T000430";"signals_accounted"=2u;"last_signal_time"="20200528T000403.486000";};
{"tag"="complete";"persistent_id"=5681202146741528770u;"segment_index"=1u;"travel_time"=1.5;"time"="20200528T000430";"signals_accounted"=2u;"last_signal_time"="20200528T000403.486000";};
{"tag"="complete";"persistent_id"=5681202146741528770u;"segment_index"=2u;"travel_time"=1.5;"time"="20200528T000430";"signals_accounted"=2u;"last_signal_time"="20200528T000404.486000";};
{"tag"="complete";"persistent_id"=5681202146741528770u;"segment_index"=3u;"travel_time"=1.5;"time"="20200528T000430";"signals_accounted"=2u;"last_signal_time"="20200528T000405.486000";};
