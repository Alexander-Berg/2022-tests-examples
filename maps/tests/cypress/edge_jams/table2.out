{"tag"="incomplete";"persistent_id"=5681202146741528770u;"travel_time"=5.82334533824;"time"="20200528T000400";"signals_accounted"=6u;"last_signal_time"="20200528T000305.486000"};
{"tag"="incomplete";"persistent_id"=5681202146741528770u;"travel_time"=5.82334533824;"time"="20200528T000430";"signals_accounted"=6u;"last_signal_time"="20200528T000405.486000"};
{"tag"="complete";"persistent_id"=5681202146741528770u;"travel_time"=6.0;"time"="20200528T000400";"signals_accounted"=8u;"last_signal_time"="20200528T000305.486000"};
{"tag"="complete";"persistent_id"=5681202146741528770u;"travel_time"=6.0;"time"="20200528T000430";"signals_accounted"=8u;"last_signal_time"="20200528T000405.486000"};
